src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct.txt

```diff
@@ -0,0 +1,59 @@
+>> Distinct(Blank(), true)
+Blank()
+
+>> Distinct([], true)
+Table()
+
+>> Distinct([{A: 1}, {A: 2}, {A: 3}, {A: 4}], true)
+Table({Result:true})
+
+>> Distinct([{A: 1}, {A: 2}, {A: 3}, {A: 4}], false)
+Table({Result:false})
```

Comment:
Should this return the empty table? since formula is always false, meaning everything would not evaluate as distinct?